lib/screen/account/access_method_page.dart

```diff
@@ -37,52 +52,65 @@ class AccessMethodPage extends StatelessWidget {
             child:
                 Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
               Text(
-                "access_method".tr(),
+                "add_account".tr(),
                 style: theme.textTheme.headline1,
               ),
               addTitleSpace(),
-              if (walletApp == 'WalletApp.MetaMask') ...[
-                Container(
-                  color: AppColor.chatPrimaryColor,
-                  child: Padding(
-                    padding: const EdgeInsets.all(10.0),
-                    child: Column(
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: [
-                        Text(
-                          'important'.tr(),
-                          style: theme.textTheme.atlasGreyBold14,
-                        ),
-                        const SizedBox(
-                          height: 5,
-                        ),
-                        RichText(
-                          text: TextSpan(
-                              text: 'autonomy_currently'.tr(),
-                              children: [
-                                TextSpan(
-                                    text: '${'ethereum_mainnet'.tr()}. ',
-                                    style: theme.textTheme.atlasBlackBold14),
-                                TextSpan(text: 'all_other_evm_networks'.tr()),
-                              ],
-                              style: theme.textTheme.atlasBlackNormal14),
-                        ),
-                      ],
-                    ),
-                  ),
-                ),
+              if (injector<ConfigurationService>().isDoneOnboarding()) ...[
```

Comment:
I think there is a rare case when `isDoneOnboarding` is false but the default account is created. could you please recheck?

Reply:
in this page, the "create new account" widget don't show up on onboarding stage. in onboarding, create new account option is in the previous step